app/views/orders/index.html.erb

```diff
@@ -19,77 +19,47 @@
 </div>
+<br>
 
-<div class="clear"></div>
 
-<div class="float-left order-container">
-  <h2>Allergies – No Problem!</h2>
-  <p>Please specify any allergies.</p>
-  <textarea rows="6" cols="67">
-  </textarea>
-  <br><br>
-</div>
-
-<div class="float-left order-container">
-  <h2>Delivery</h2>
-  Please specify details for delivery or pick-up. <br>
-  <form action="">
-    <input type="radio" name="delivery" value="pick_up" class="radio_style"> We’d like to pick up the order <br> ourselves at Tegelbacken 4A. <br>
-    <input type="radio" name="delivery" value="deliver" class="radio_style"> We'd like the order to be delivered <br> directly to us by cargo bike.
-  </form>
-  <br>
-  Delivery date: <br>
-  <input type="date" name="delivery[date]" id="delivery_date" value="" placeholder="Date"
-         size="5" maxlength="25" style="width: 374px;"> <br>
-  <input type="time" name="delivery[time]" id="delivery_time" value="" placeholder="Time"
-         size="5" maxlength="25" style="width: 374px;"> <br>
+<%= form_for @order do |f| %>
 
-</div>
+  <div class="float-left order-container">
+    <h4>Delivery Information:</h4>
+    <%= f.date_field :delivery_date, placeholder: 'Delivery Address' %>
+    <%= f.text_field :delivery_method, placeholder: 'Delivery Method' %>
+  </div>
+  <div class="float-left order-container">
+    <h4>Delivery address:</h4>
+    <%= f.text_field :delivery_name, placeholder: 'Delivery Name' %>
+    <%= f.text_field :delivery_address, placeholder: 'Delivery Address' %>
+    <%= f.text_field :delivery_postal_code, placeholder: 'Delivery Postal Code' %>
+    <%= f.text_field :delivery_city, placeholder: 'Delivery City' %>
+    <%= f.text_field :delivery_floor, placeholder: 'Delivery Floor' %>
+    <%= f.text_field :delivery_door_code, placeholder: 'Deliovery Door Code' %>
+    <%= f.text_field :delivery_contact_name, placeholder: 'Delivery Contact Name' %>
+    <%= f.text_field :delivery_contact_phone_number, placeholder: 'Delivery Contact Phone' %>
+  </div>
+  <div class="float-left order-container">
+    <h4>Billing address:</h4>
+    <%= f.text_field :billing_name, placeholder: 'Billing Name' %>
+    <%= f.text_field :billing_company, placeholder: 'Billing Company' %>
+    <%= f.text_field :billing_org_nr, placeholder: 'Billing Organisation Number' %>
+    <%= f.text_field :billing_address, placeholder: 'Billing Address' %>
+    <%= f.text_field :billing_postal_code, placeholder: 'Billing Postal Code' %>
+    <%= f.text_field :billing_city, placeholder: 'Billing City' %>
+    <%= f.text_field :billing_phone, placeholder: 'Billing Phone Number' %>
+    <%= f.text_field :billing_email, placeholder: 'Billing email' %>
+  </div>
+  <div class="float-left order-container">
+    <h2>Allergies – No Problem!</h2>
+    <h4>Please specify any allergies:</h4>
+    <%= f.text_field :allergies, placeholder: 'Allergies' %>
+  </div>
+  <div class="float-left order-container">
+    <%= f.text_field :boxes, placeholder: 'Boxes' %>
```

Comment:
I believe that boxes filed should be placed early in the form, otherwise it might be forgotten.

Reply:
The should not be a part of the form. I will add it as a task in the remodeling of the look and feal of the form.

Reply:
ok

Reply:
I will remove the box.x I should not be in the view. Only in the admin interface. Will do that in a new PR there I work on look and feel of the form.